tests/test_streaming_language_modeling_task.py

```diff
@@ -68,7 +74,7 @@ def get_last_log_stats_containing_string(log_records, search_string):
                     task="streaming_language_modeling",
                     max_tokens=None,
                 )
-            train_log = get_last_log_stats_containing_string(logs.records, "train_loss")
+            train_log = get_last_log_stats_containing_string(logs.records, '"loss"')
```

Comment:
Was this test not running before? How did these log lines all get renamed...

Reply:
Yeah, it doesn't run as part of CI/actions builds (the test_streaming_language_modeling_task.py under cpu_tests/ is run, but not this one, and they're different). The logs produced here don't contain 'train_loss' (anymore).